src/molecule.cpp

```diff
@@ -29,6 +29,10 @@
 #include <cmath>
 #include <cstring>
 
+#include <fstream>
+#include <iostream>
+
+using namespace std;
```

Comment:
Please remove `using namespace std;` and instead use the `std::` namespace prefix where needed. That helps avoiding ambiguities (there are lots of classes and symbols in the `std` namespace, some with common names that may conflict with what LAMMPS is using).

That said, selected imports like with `using xxx = std::xxx;` are acceptable, if there are no ambiguities with LAMMPS classes. Keep in mind that all "style headers" are included at the same time, so the risk for namespace clashes is high and we have to be careful and the more careful the larger LAMMPS gets. :-(